content/components/recipes/DisableRecipeButton.tsx

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import { Alert, Button, Icon, IconButton, Modal, SelectPicker } from "rsuite";
+
+import { useSelectedNormandyEnvironmentAPI } from "devtools/contexts/environment";
+import {
+  ACTION_UPDATE_DATA,
+  useRecipeDetailsDispatch,
+} from "devtools/contexts/recipeDetails";
+import { Revision } from "devtools/types/recipes";
+
+const ENDING_REASONS = [
+  "Experiment Complete",
+  "Filtering Issue",
+  "Enrollment Issue",
```

Comment:
How are "Filtering Issue" and "Enrollment Issue" different? It might be good to explain what these different categories mean, so we can be consistent about them.